_layouts/default.html

```diff
@@ -733,7 +733,7 @@
     <body class="home page-template-default page page-id-6 custom-background">
 
 <!--Select includes/main_nav according to page lang or absence of lang-->
-          {% if page.lang == "pt-PT"  %}
+          {% if page.lang == "de" or page.lang == "es" or page.lang == "pt-PT" or page.lang == "zh-CN" %}
```

Comment:
Is it possible to do this? To avoid adding languages to the condition every time...with lots of languages, this will quickly get unwieldy.

if page.lang == "en""
    include main_nav
else 
    include main_nav_tr

Reply:
I reworked it to look like this:
          {% if page.lang == "en" or page.ref == "blog" %}
            {% include main_nav.html %}
          {% else %}
            {% include main_nav_tr.html %}
          {% endif %}

Otherwise the Blog page doesn't render a nav bar.